### PR TITLE
Update contributors packs data

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/29189

## Description
Update support data for some contributors packs.

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@barchen1 is this ready for review?

**Comment by barchen1** [comment]
> @barchen1 is this ready for review?

Yes, I went throw on all of the packs in the issue and change the pack_metadata accordingly.

File: Packs/Aella_StarLight/pack_metadata.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "Aella Star Light",
   "description": "Aella Star Light Integration",
-  "support": "partner",
-  "currentVersion": "1.0.0",
+  "support": "community",
```
Comment (ShahafBenYakir) [inline Packs/Aella_StarLight/pack_metadata.json:4]: @marcellarichmond @mchasepan Why have we decided to change `Aella Star Light` to community and not XSOAR?

File: Packs/Genians/pack_metadata.json [hunk 2]
```diff
@@ -2,9 +2,9 @@
     "name": "Genians",
     "description": "Use the Genians integration to block IP using assign tag and unassign tag.",
     "support": "partner",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.0.1",
     "author": "Genians",
-    "url": "https://docs.genians.com/release/en/",
+    "url": "",
```
Comment (ShahafBenYakir) [inline Packs/Genians/pack_metadata.json:7]: @mchasepan What about this: https://www.genians.com/resources-overview/

File: Packs/Luminate/pack_metadata.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "Luminate",
     "description": "Enrich your report and Respond to incidents with Luminate",
-    "support": "partner",
-    "currentVersion": "1.0.0",
+    "support": "community",
```
Comment (ShahafBenYakir) [inline Packs/Luminate/pack_metadata.json:4]: @marcellarichmond @mchasepan Why have we decided to change Aella Star Light to community and not XSOAR?

File: Packs/Luminate/pack_metadata.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+    "currentVersion": "1.0.1",
     "author": "Luminate",
     "url": "https://www.paloaltonetworks.com/cortex",
```
Comment (ShahafBenYakir) [inline Packs/Luminate/pack_metadata.json:7]: If this goes to the community we need to remove it, if we change to XSOAR then keep it.

File: Packs/PassiveTotal/pack_metadata.json [hunk 5]
```diff
@@ -2,10 +2,10 @@
     "name": "PassiveTotal",
     "description": "Analyze and understand threat infrastructure from a variety of sources\u2013passive DNS, active DNS, WHOIS, SSL certificates and more\u2013without devoting resources to time-intensive manual threat research and analysis",
     "support": "partner",
-    "currentVersion": "2.0.2",
+    "currentVersion": "2.0.3",
     "author": "RiskIQ",
     "url": "https://www.riskiq.com",
```
Comment (ShahafBenYakir) [inline Packs/PassiveTotal/pack_metadata.json:7]: @marcellarichmond @mchasepan We need to replace this, what about this one: https://www.riskiq.com/resources/support/

File: Packs/RiskIQDigitalFootprint/pack_metadata.json [hunk 6]
```diff
@@ -2,10 +2,10 @@
     "name": "RiskIQ Digital Footprint",
     "description": "RiskIQ Digital Footprint integration enables your security team to manage assets outside your firewall and provides you with the ability to add or update assets and analyze your digital footprint from the view of the global adversary.",
     "support": "partner",
-    "currentVersion": "1.0.2",
+    "currentVersion": "1.0.3",
     "author": "RiskIQ",
     "url": "https://www.riskiq.com",
```
Comment (ShahafBenYakir) [inline Packs/RiskIQDigitalFootprint/pack_metadata.json:7]: @marcellarichmond @mchasepan Same as passive total

File: Packs/SecurityIntelligenceServicesFeed/pack_metadata.json [hunk 7]
```diff
@@ -2,10 +2,10 @@
     "name": "Security Intelligence Services Feed",
     "description": "A PassiveTotal with Security Intelligence Services Feed can provide you newly observed Domain, Malware, Phishing, Content and Scam Blacklist.",
     "support": "partner",
-    "currentVersion": "1.0.1",
+    "currentVersion": "1.0.2",
     "author": "RiskIQ",
     "url": "https://www.riskiq.com",
```
Comment (ShahafBenYakir) [inline Packs/SecurityIntelligenceServicesFeed/pack_metadata.json:7]: @marcellarichmond @mchasepan Same as passive total

File: Packs/Silverfort/pack_metadata.json [hunk 8]
```diff
@@ -2,9 +2,9 @@
     "name": "Silverfort",
     "description": "Silverfort protects organizations from data breaches by delivering strong authentication across entire corporate networks and cloud environments, without requiring any modifications to endpoints or servers. Using patent-pending technology, Silverfort's agentless approach enables multi-factor authentication and AI-driven adaptive authentication even for systems that don\u2019t support it today, including proprietary systems, critical infrastructure, shared folders, IoT devices, and more. Use Silverfort integration to get & update Silverfort risk severity. This integration was integrated and tested with Silverfort version 2.12.",
     "support": "partner",
-    "currentVersion": "1.0.2",
+    "currentVersion": "1.0.3",
     "author": "Silverfort",
-    "url": "Silverfort.com",
+    "url": "silverfort.com",
```
Comment (ShahafBenYakir) [inline Packs/Silverfort/pack_metadata.json:7]: @marcellarichmond @mchasepan We need to replace this, what about this one: https://support.silverfort.com/

File: Packs/Aella_StarLight/pack_metadata.json (hunk 1 above)
Comment (mchasepan) [reply Packs/Aella_StarLight/pack_metadata.json:4]: Per @NeelRust

File: Packs/Luminate/pack_metadata.json (hunk 3 above)
Comment (mchasepan) [reply Packs/Luminate/pack_metadata.json:4]: Per @NeelRust

File: Packs/PassiveTotal/pack_metadata.json (hunk 5 above)
Comment (mchasepan) [reply Packs/PassiveTotal/pack_metadata.json:7]: approved

File: Packs/RiskIQDigitalFootprint/pack_metadata.json (hunk 6 above)
Comment (mchasepan) [reply Packs/RiskIQDigitalFootprint/pack_metadata.json:7]: yes plz

File: Packs/SecurityIntelligenceServicesFeed/pack_metadata.json (hunk 7 above)
Comment (mchasepan) [reply Packs/SecurityIntelligenceServicesFeed/pack_metadata.json:7]: yes plz

File: Packs/Genians/pack_metadata.json (hunk 2 above)
Comment (mchasepan) [reply Packs/Genians/pack_metadata.json:7]: ok, approved.

File: Packs/Luminate/pack_metadata.json (hunk 4 above)
Comment (mchasepan) [reply Packs/Luminate/pack_metadata.json:7]: Remove URL

File: Packs/Silverfort/pack_metadata.json (hunk 8 above)
Comment (mchasepan) [reply Packs/Silverfort/pack_metadata.json:7]: Approved

**Review by danikdanik — APPROVED** [review]

File: Packs/Genians/pack_metadata.json (hunk 2 above)
Comment (barchen1) [reply Packs/Genians/pack_metadata.json]: done

File: Packs/Luminate/pack_metadata.json (hunk 4 above)
Comment (barchen1) [reply Packs/Luminate/pack_metadata.json]: done

File: Packs/PassiveTotal/pack_metadata.json (hunk 5 above)
Comment (barchen1) [reply Packs/PassiveTotal/pack_metadata.json]: done

File: Packs/RiskIQDigitalFootprint/pack_metadata.json (hunk 6 above)
Comment (barchen1) [reply Packs/RiskIQDigitalFootprint/pack_metadata.json]: done

File: Packs/SecurityIntelligenceServicesFeed/pack_metadata.json (hunk 7 above)
Comment (barchen1) [reply Packs/SecurityIntelligenceServicesFeed/pack_metadata.json]: done

File: Packs/Silverfort/pack_metadata.json (hunk 8 above)
Comment (barchen1) [reply Packs/Silverfort/pack_metadata.json]: done

**Review by ShahafBenYakir — APPROVED** [review]

File: Tests/conf.json [hunk 9]
```diff
@@ -3161,7 +3161,8 @@
         "Code42-Test": "Issue 29093",
         "SMB test": "Issue 26454",
         "TestCofenseFeed": "Issue 29198",
-        "PAN-OS Query Logs For Indicators Test": "Issue 28753"
+        "PAN-OS Query Logs For Indicators Test": "Issue 28753",
+        "Indeni_test": "Failed on Max retries, skipping in order to merge a PR"
```
Comment (ShahafBenYakir) [inline Tests/conf.json:3165]:
```suggestion
        "Indeni_test": "Non-certified partner pack, this should be skipped for now"
```

Comment (ShahafBenYakir) [reply Tests/conf.json:3165]: You can just commit this change

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**